content/updates/2017-08-11-continuous-improvement-more-tools-help-use-cloud-gov.md

```diff
@@ -0,0 +1,56 @@
++++
+date = "2017-08-11"
+title = "Quarterly update: Continuous improvement, more tools to help you use cloud.gov"
++++
+
+Every quarter we update you with the important news about new features and updates to the cloud.gov platform. In this edition of the newsletter we have updates about the cloud.gov dashboard and other features for our users as well as stories from our customers, and links to recent presentations about cloud.gov.
+
+### What’s new
+
+#### Use the dashboard to invite new users to your team
+
+Managing users in your organization and granting them the permissions they need should be a straightforward process so your team can get to work building your app on cloud.gov. To make it easier for you, we built new features for inviting new users and giving them roles within your organization using the dashboard. With these changes, even the least technical people on your team can invite new folks and manage their role without involving cloud.gov support.
+
+<img src="/img/invite-users.png" alt="The user invitation screen with notification" width="624" height="272" />
+
+Being able to manage your users from the dashboard will also allow you to have finer control over who has access to different parts of your system without having to involve your technical staff. Our users requested this functionality, so if you get a chance to use it please let us know how it’s working for you. Watch for more developments like this in the future.
```

Comment:
We removed this second paragraph (and adjusted the earlier one) a bit [in the update sent out](http://us9.campaign-archive2.com/?u=6f1977de9eff4c384dc8d6527&id=fbc1e98d98) - fine with me to bring it back if you'd like, just not sure if intentional. (Also _just_ noticed a typo in the update we sent - "With these changes, it's easy for even the least technical people on your team **can** invite new folks and manage their roles.")